dotCMS/src/main/java/com/dotmarketing/db/HibernateUtil.java

```diff
@@ -76,12 +87,12 @@ protected TransactionListenerStatus initialValue() {
     };
 
 	private static final ThreadLocal<Boolean> asyncCommitListenersFinalization = ThreadLocal.withInitial(()->true);
-	
-	private static final ThreadLocal< Map<String,DotRunnable> > commitListeners=new ThreadLocal<Map<String,DotRunnable>>() {
-	    protected java.util.Map<String,DotRunnable> initialValue() {
-	        return new LinkedHashMap<String,DotRunnable>();
-	    }
-	};
+
+	private static final ThreadLocal<Map<String, DotRunnable>> asyncCommitListeners = ThreadLocal
+			.withInitial(() -> new LinkedHashMap<String, DotRunnable>());

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)